src/resources/Incorporations/limitedCompany.ts

```diff
@@ -80,7 +80,7 @@ export const LimitedCompanyResource: IncorporationResourceIF = {
         ]
       }
     ],
-    article: 'Sample_articles.pdf',
+    article: 'bc_limited_company_corporation_agreement.pdf',
```

Comment:
Added as a place holder until the new pdf is created. Requested by UX assurance.